R/truncnorm.R

```diff
@@ -19,10 +19,12 @@ my_etruncnorm= function(a,b,mean=0,sd=1){
   #Fix a bug of quoting the truncnorm package
   #E(X|a<X<b)=a when a==b is a natural result
   #while etruncnorm would simply return NaN,causing PosteriorMean also NaN
+  # ZMZ:  when a and b are both negative and far from 0, etruncnorm can't compute
+  # the mean and variance. Also we should deal with 0/0 situation caused by sd = 0.
   tmp1=etruncnorm(alpha,beta,0,1)
   isequal=is.equal(alpha,beta)
   
-  tmp1[isequal]=alpha[isequal]
+  tmp1[isequal]=beta[isequal]
```

Comment:
it's necessary. Because changing alpha to beta can simultaneously deal with alpha==beta and alpha=-Inf, beta=-10000 cases. If we use alpha instead, the second case will produce wrong output.